Gemfile.lock

```diff
@@ -351,4 +351,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.15.1
+   1.15.4
```

Comment:
We seem to be using `1.15.1` on the VMs, so maybe skip this change. I think we'll keep changing it back & forth otherwise?

https://github.com/alphagov/govuk-puppet/search?q=bundler_version